exportify.js

```diff
@@ -319,16 +319,16 @@ var PlaylistExporter = {
       var tracks = responses.map(function(response) {
         return response.items.map(function(item) {
           return [
-            item.track.uri,
             item.track.name,
             item.track.artists.map(function(artist) { return artist.name }).join(', '),
+            item.track.artists.map(function(artist) { return artist.name }).join(', '),
             item.track.album.name,
             item.track.disc_number,
             item.track.track_number,
             item.track.duration_ms,
-            item.added_by == null ? '' : item.added_by.uri,
+            item.added_at,
```

Comment:
Is there a purpose behind having item.added_at being listed twice here? Why not just remove this duplicate if added_by isn't supported by iTunes? Or is this just a typo?